spectator-web-spring/build.gradle

```diff
@@ -15,7 +15,7 @@
  */
 
 dependencies {
-  compileApi project(':spectator-api')
+  compile project(':spectator-api')
```

Comment:
Hmm, I was actually wondering if the spring dependencies should be compileApi because the annotations from them are part of the public interface of some of the classes. I'm pretty sure the spectator-api dependency was correct as compileApi.

I don't know how familiar you are with compile vs compileApi, but the basic difference is compileApi will cause them to go into the published POM file as a compile scope dependency where compile will cause them to go into the POM as a runtime scope dependency (more background in https://github.com/nebula-plugins/nebula-publishing-plugin/issues/31).